### PR TITLE
Move preview API into core

### PR DESCRIPTION
**Pull request by fpagnoux**
Connected to openfisca/openfisca-web-api#129

## 12.1.0

* Include the preview API into OpenFisca-Core
* Remove internationalization (`i18n`, `Babel`)
* Use CircleCI instead of Travis

### CODE REVIEW

**Comment by fpagnoux** [comment]
[Proof that the continuous releasing is working](https://circleci.com/gh/openfisca/openfisca-core/28)

File: README.md [hunk 1]
```diff
@@ -1,23 +1,37 @@
...
+## Environment
 
-The documentation of the project is hosted at https://doc.openfisca.fr/
+This package requires Python 2.7
 
-## Install for development
+## Installation
 
-This is the way to install OpenFisca-Core for development. If you just want to modify the legislation formulas and/or parameters, just follow the [installation section](https://doc.openfisca.fr/install.html) of the documentation.
+If you're developping your own country package, you don't need to explicitely install OpenFisca-Core. It just needs to appear [in your package dependencies](https://github.com/openfisca/openfisca-france/blob/18.2.1/setup.py#L53).
```
Comment (cbenz) [inline README.md]: explicitely -> explicitly

File: README.md [hunk 2]
```diff
@@ -1,23 +1,37 @@
...
 
-This package requires Python 2.7.
+If you want to contribute to OpenFisca-Core itself, welcome! To install it locally in developpment mode :
```
Comment (cbenz) [inline README.md]: No space before `:`

File: README.md [hunk 3]
```diff
@@ -1,23 +1,37 @@
...
+
+## Testing
+
+```sh
+make test
+```
+
+## Serving the API
+
+OpenFisca-Core provides a Web-API. To run it with the mock country package `openfisca_dummy_country`, run:
```
Comment (cbenz) [inline README.md]: How can I install this dummy country? Could you add this information?

File: circle.yml [hunk 4]
```diff
@@ -0,0 +1,24 @@
+machine:
+  python:
+    version: 2.7.12
```
Comment (cbenz) [inline circle.yml]: > Is `.12` really necessary?

File: setup.py [hunk 5]
```diff
@@ -35,21 +34,24 @@
...
             'openfisca-dummy-country == 0.1.6',
             ],
         },
     include_package_data = True,  # Will read MANIFEST.in
     install_requires = [
-        'Babel >= 0.9.4',
         'Biryani[datetimeconv] >= 0.10.4',
         'numpy >= 1.11',
         'PyYAML >= 3.10',
+        'flask == 0.12',
```
Comment (cbenz) [inline setup.py]: Should we introduce an extra requirement `[web_api]`?

File: circle.yml (hunk 4 above)
Comment (fpagnoux) [inline circle.yml]: Yes, the build is [really slow](https://circleci.com/gh/openfisca/openfisca-core/8) otherwise.

File: setup.py (hunk 5 above)
Comment (fpagnoux) [inline setup.py]: Don't know. The advantage of not making it a extra require is that we can do:
```sh
pip install openfisca_france
COUNTRY_PACKAGE=openfisca_france gunicorn "openfisca_web_api.app:create_app()" --bind localhost:5000 --workers 3
```

Comment (fpagnoux) [inline setup.py]: Is this just a suggestion or a required change?

I thought about it, but it makes everything more complex and I don't see the real use case. The main drawback here is that we make the installation heavier, but the coast is quite low.

File: README.md (hunk 3 above)
Comment (fpagnoux) [inline README.md]: It has already been installed with `pip install -e "[test]"`.

Comment (fpagnoux) [inline README.md]: Suggestions are welcome if you think it's worth expliciting where the package comes from.

**Pull request merged**